awscrt/s3.py

```diff
@@ -696,6 +695,11 @@ def _on_finish(
             self._finished_future.set_exception(error)
         else:
             self._finished_future.set_result(None)
+        if checksum_validation_algorithm:
```

Comment:
utterly trivial: newline between previous if/else block and new if/else block
```suggestion

        if checksum_validation_algorithm:
```